jsonnet/utils/set-terminationMessagePolicy.libsonnet

```
@@ -0,0 +1,42 @@
+{
+  setTerminationMessagePolicy(o): o {
+    local addTerminationMessagePolicy(o) = o {
+      [if std.setMember(o.kind, ['DaemonSet', 'Deployment', 'ReplicaSet']) then 'spec']+: {
+        template+: {
```

this + using `spec` instead of `super` seems to work.
```suggestion
        template+: {
                local spec = super.spec,
```

---

This works, though the same doesn't seem to work for the `Alertmanager` ... kinds. However just referencing `o.spec` or `o.spec.template.spec` seems to work well.